node/src/main/scala/coop/rchain/node/ReplClient.scala

```diff
@@ -25,6 +29,20 @@ class GrpcReplService(host: String, port: Int) extends ReplService[Task] {
   }
 
   def eval(fileName: String): Task[String] = Task.delay {
-    blockingStub.eval(EvalRequest(fileName)).output
+    val file = new File(fileName)
+    if (file.exists()) {
+      using(Source.fromFile(file)) { source =>
+        blockingStub.eval(EvalRequest(source.getLines.mkString)).output
+      }
+    } else {
+      s"File $fileName not found"
+    }
   }
+
+  private def using[S <: Closeable, A](source: S)(use: S => A): A =
```

Comment:
This seems pretty general, maybe worth putting the `shared` project

Reply:
Over in rspace-land, we have:

https://github.com/rchain/rchain/blob/91b9423f8b54d1ddba9eabda4597a0d10afa4cd0/rspace/src/main/scala/coop/rchain/rspace/util/package.scala#L17-L31

Reply:
yes, Henry's solution is even more general. I will create a copy in shared. Rspace can then move to the shared version when ready.